### DISCUSSION TITLE
Matching image snapshot fails with cy.screenshot() failed because it requires a DOM element

### DISCUSSION THREAD

**Discussion by Fastova** [simonsmith/cypress-image-snapshot#40] [General]
Hi guys, thank you for your great tool!

I'm facing a problem currently, which I can't overgo. Could you please advise, if I'm missing smth?

I"ve installed a plugin,
configured it and added a command to cypress.config 
```
import { addMatchImageSnapshotPlugin } from '@simonsmith/cypress-image-snapshot/plugin';

addMatchImageSnapshotPlugin(on);
```

Added types for the plugin in ts.config:
`@simonsmith/cypress-image-snapshot/types`
 
In my test the command is available, it runs fine:
```
cy.matchImageSnapshot('snapshot', {
        failureThreshold: 0.4,
        blur: 10,
        capture: 'viewport',
        diffFilenameExtension: '.wrong',
      });
```

However it fails with no DOM element:

```
get #el-id
wrap<div#el-id>
taskMatching image snapshot, Object{12}
screenshotstatistic-card, Object{12}

CypressError
cy.screenshot() failed because it requires a DOM element.

The subject received was:
> null

The previous command that ran was:
> cy.task()

All 3 subject validations failed on this subject.
```

Am I doing smth wrong?
Regular cy.screenshot() works and saves the screenshot.

**Comment by simonsmith** [comment]
Moved to #45
